src/web/jcc/client/components/CommandControl/CommandControl.tsx

```diff
@@ -1054,14 +1054,11 @@ export default class CommandControl extends React.Component {
                     if (!this.state.taskPacketsTimeline.keepEndDateCurrent) {
                         end = this.state.taskPacketsTimeline.end as string;
                     }
-                    this.api
-                        .getTaskPackets(this.state.taskPacketsTimeline.start as string, end)
-                        .then((taskPackets) => {
-                            this.setTaskPackets(taskPackets);
-                            taskData.updateTaskPacketsLayers(taskPackets);
-                        })
-                        .catch((err) => {
-                            console.error("Task Packets Retrieval Error:", err);
+
```

Comment:
Since this function was modified and does not have a function comment yet we should add one.